src/options/README

```diff
@@ -0,0 +1,225 @@
+Modules
+=======
+
+  Each options module starts with the following required attributes:
+
+    id     ... (string) ID of the module (e.g., "ARITH")
+    name   ... (string) name of the module (e.g., "Arithmetic Theory")
+    header ... (string) name of the options header to generated (e.g., "options/arith_options.h")
+
+  A module defines 0 or more options and/or aliases.
+
+  In general, each attribute/value pair is required to be in one line.
+  Comments start with # and are not allowed in attribute/value lines.
+
+
+Options
+=======
+
+  Options can be defined with the [[option]] tag, the required attributes for
+  an option are:
+
+    category ... (string) common | expert | regular | undocumented
+    type     ... (string) C++ type of the option value
+
+  Optional attributes are:
+
+    name       ... (string) option name that is used to access via
+                            options::<name>()
+    smt_name   ... (string) alternative name to access option via
+                            set-option/get-option commands
+    short      ... (string) short option name consisting of one character
+                            (no - prefix required)
```

Comment:
Maybe this would be more readable:
`(no '-' prefix required)`

Reply:
Done